src/components/Blog-Preview.js

```diff
@@ -2,37 +2,48 @@ import React from 'react';
 import { Link } from 'gatsby';
 import PropTypes from 'prop-types';
 import frontmatter from './prop-types/frontmatter';
+import previewStyle from './styles/post-preview.module.css';
 
-export default function BlogPreview(props) {
-  const { postInfo } = props;
+export default function BlogPreview({ postInfo }) {
+  const styles = previewStyle;
+  // REFACTOR
+  const tags = [];
+  for (let i = 0; i < 3; i += 1) {
+    if (postInfo.frontmatter.tags[i]) {
+      tags.push(`#${postInfo.frontmatter.tags[i]}`);
+    } else {
+      tags.push('');
```

Comment:
Hmm, what do tags do when they're just a blank string like this? If you're just trying to display the first three tags, it might be cleaner to use some array methods, like so:

```js
const tags = postInfo.frontmatter.tags.slice(0, 3).map(({ tag }) => `#tag`)
```